functions/Get-DbaServerInstallDate.ps1

```diff
@@ -0,0 +1,181 @@
+function Get-DbaServerInstallDate
+{
+<#
+.SYNOPSIS
+Returns the install date of SqlServer and Windows Server, depending on what is passed. 
+	
+.DESCRIPTION
+By default, this command returns for each SQL Server instance passed in:
+SQL Instance install date, formatted as a string
+Hosting Windows server install date, formatted as a string
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER SqlCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER WindowsCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER SqlOnly
```

Comment:
I think we've done something similar with DB's or accounts:
-NoSystemDB or -IncludeServerLevel are examples where there also are only 2 categories.
Some started with -...Only, but I think we put it in the 100 commandments to use -Include... or -Exclude... or -No....